sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -48,26 +53,16 @@
  * @see EventHubProducerAsyncClient
  */
 public class EventData {
-    /*
-     * These are properties owned by the service and set when a message is received.
-     */
-    static final Set<String> RESERVED_SYSTEM_PROPERTIES;
 
-    private final Map<String, Object> properties;
-    private final BinaryData body;
-    private final SystemProperties systemProperties;
-    private Context context;
+    private static final int MAX_MESSAGE_ID_LENGTH = 128;
+    private static final int MAX_PARTITION_KEY_LENGTH = 128;
+    private static final int MAX_SESSION_ID_LENGTH = 128;
```

Comment:
EH has no session

Reply:
The sessionId in service bus leverages the AMQP concept of a groupId, I believe this should be renamed to use GroupID instead

Reply:
According to your comment, remove unnecessary parameters, variables and methods in the new version